parlai/torchscript/modules.py

```diff
@@ -750,6 +755,188 @@ def utf8_chars(self, s: str) -> List[str]:
         return chars
 
 
+@torch.jit.script
+class ScriptableSubwordBpeHelper(object):
+    """
+    Version of parlai.utils.bpe.ScriptableSubwordBpeHelper that can be TorchScripted.
+    """
+    @classmethod
+    def findall(cls, text: str) -> List[str]:
+        """
+        Split tokens in a manner that replicates parlai.utils.bpe.Gpt2BpeHelper.
+        """
+        tokens: List[str] = []
+        idx = 0
+        num_passes = 0
+        while idx < len(text):
+            num_passes += 1
+            if num_passes > 10000:
+                return ["*** Infinite loop in ScriptableSubwordBpeHelper.findall()! ***"]
+            if not text[idx].isspace():
+                # Capture runs of one type of character
+                if text[idx] == " ":
+                    last_matching_idx = idx + 1
+                else:
+                    last_matching_idx = idx
+                if text[last_matching_idx].isalnum() or text[last_matching_idx] == '_':
```

Comment:
can we abstract a way a helper method that's shorthand for isalnum() or == '_' to improve readability?

Reply:
Added helper function